src/2a_analyze_MPA.jl

```diff
@@ -0,0 +1,100 @@
+"""Identify suitable locations for each region."""
+
+include("common.jl")
+
+@everywhere begin
+    """
+        prop_suitable(subsection::AbstractMatrix)::Float32
+
+    Calculate the the proportion of the subsection that is suitable for deployments.
+    Subsection is the surrounding hectare centred on each cell of a raster.
+    """
+    function prop_suitable(subsection::AbstractMatrix)::Int16
+        total = sum(subsection)
+        if total == 0.0
+            return 0.0
+        end
+
+        return Int16(round((total / length(subsection))*100))
+    end
+
+    function _write_data(fpath::String, data, cache)::Nothing
+        if !isfile(fpath)
+            if occursin("grouped_", fpath)
+                cache .= label_components(data)
+            else
+                cache .= data
+            end
+
+            write(fpath, cache; force=true)
+        end
+
+        return nothing
+    end
+
+    function assess_region(reg)
+        # Load prepared bathymetry raster
+        src_bathy = Raster(joinpath(MPA_OUTPUT_DIR, "$(reg)_bathy.tif"); crs=EPSG(7844), lazy=true)
```

Comment:
Could this comment repetition be avoided and expressed once above all of these Raster reads